harmoni_actuators/harmoni_web/scripts/harmoni_web/web_service.py

```diff
@@ -70,19 +73,44 @@ def do(self, data):
         data = super().do(data)
         self.state = State.REQUEST
         self.actuation_update(actuation_completed=False)
+        data_array = self._get_web_data(data)
         try:
             rospy.sleep(1)
-            self._send_request(data)
+            for data in data_array:
+                self.send_request(data)
+                rospy.sleep(0.2)
             self.state = State.SUCCESS
             self.actuation_update(actuation_completed=True)
         except:
             self.state = State.FAILED
             self.actuation_update(actuation_completed=True)
         return
 
-    def _send_request(self, display_view):
+    def _get_web_data(self, data):
+        data = ast.literal_eval(data)
```

Comment:
Should this be put into a "try" in case data is malformed (possible with bad WiFi)? If data correctness is already checked before here, ignore this comment.

Reply:
I will push this to @micolspitale93 as well